### DISCUSSION TITLE
Auto pausing after a time

### DISCUSSION THREAD

**Discussion by Aermoe** [winder/Universal-G-Code-Sender#1885] [Help]
Hello

I have added a laser module for cutting/ carving to my CNC machine and use it through UGS from a raspberry pi.
The laser module can only run for 30 min at a time (advised by the manufacturer). LaserGRBL has a feature to pause the machine after a certain amount of running time. Is this also possible in UGS? If so how can it be done? I would like the laser to auto pause/ hold after 25min and continue after a 2 or 3 min cooldown.

Thanks in advance!

**Comment by breiler** [comment]
No there is no such feature. 

Is there anyone else that would like to see such a feature, is it worthwhile for us to implement it?

**Comment by Aermoe** [comment]
I suspect I'm not the only one running a laser with UGS? :)
